src/shared/compounders/AMMCompounderBase.sol

```diff
@@ -60,7 +58,9 @@ abstract contract AMMCompounderBase is ReentrancyGuard, ERC4626 {
     address[] public rewardAssets;
     /// @notice The underlying assets.
     address[] public underlyingAssets;
-
+    /// @notice The internal accounting of AUM.
+    uint256 internal totalAUM;
```

Comment:
Thinking about monitoring, can this kind of information be used by e.g. Dune so we can have public Fortress dashboards to show the evolution of AUM over time? Not sure right now if `totalAUM` is readable from the outside and I am also not sure what Dune for instance needs in order to query/display on chain information. Maybe just good for us to keep an eye on how certain information are meant to be exposed for on chain metrics.

Reply:
Certainly. `totalAUM` is readable using the `totalAssets` method.